torch/csrc/jit/script/compiler.cpp

```diff
@@ -1597,10 +1597,39 @@ struct to_ir {
   // else
   //   raise Exception("Hi")
   // print(a)
-  void emitRaise(const SourceRange& loc) {
+  void emitRaise(const Raise& stmt) {
+    if (stmt.expr().present()) {
+      auto exceptionNode = stmt.expr().get().tree();
+      if (exceptionNode->kind() == TK_APPLY &&
+          !Apply(exceptionNode).inputs().empty()) {
+        auto apply = Apply(exceptionNode);
+        if (apply.callee().kind() == TK_VAR) {
```

Comment:
Would be nice to have a comment here describing what the emitted code looks like so someone scanning the code doesn't have to squint so hard :)